.github/workflows/trigger-release-branch-validation.yaml

```diff
@@ -0,0 +1,36 @@
+name: Trigger Validation Tests on Release Branch Cut
+
+on:
+  create:
+
+jobs:
+  check-branch-prefix:
```

Comment:
I haven't tested this. Would something like this work in forge-stable workflow?
```
on:
  create:
    branches:
      - aptos-release-*

...

jobs:
  determine-test-metadata:
    runs-on: ubuntu-latest
    if: github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || startsWith(github.ref_name, 'aptos-release-')


```
That way we can avoid triggering these steps for all branches

Reply:
Unfortunately, GH doesn't support such filters for create. That's why.

https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#create

Reply:
That's a bummer. Does adding `if: github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || startsWith(github.ref_name, 'aptos-release-')` work as an alternative?

Reply:
The problem with that is we will end up with a lot of workflow invocations that would just skip for every branch that is pushed. That would pollute the Forge Stable run UI, so I was trying to avoid that way. The trigger workflow, I thought, nobody would look unless someone is trying to debug something.

Reply:
That makes sense. I was a bit concerned that `github.rest.actions.createWorkflowDispatch` might make it unclear what triggered the forge-stable run but I think keeping the list of Forge stable runs clean is more important